nipype/pipeline/engine/base.py

```diff
@@ -36,11 +36,11 @@ def __init__(self, name=None, base_dir=None):
 
         """
         self._hierarchy = None
-        self._name = None
+        self.name = name
```

Comment:
Doesn't this break the property definition below? (line 46)

Reply:
why? Before we had two lines: the first one  `self._name = None` and 3 lines later `self.name = name`. I thought that one was left there forgotten.